<launch>
	<!-- main -->
	<node pkg="msg_conversion" type="image_64fc1_to_16uc1_offline" name="image_64fc1_to_16uc1_offline" output="screen">
		<param name="load_rosbag_path" type="string" value="$(env HOME)/rosbag/sq2_velodyne/dkan_outdoor/2018-10-31-08-37-11_converted_transformed_projected.bag"/>
		<!-- <param name="debug_hz" type="double" value="10.0"/> -->
		<param name="topic_0" type="string" value="/camera/color/image_raw/compressed/depth"/>
	</node>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find msg_conversion)/rviz_config/demo.rviz"/>
</launch>